compositor_pipeline/src/pipeline/decoder/audio.rs

```diff
@@ -20,22 +20,6 @@ mod fdk_aac;
 mod opus;
 mod resampler;
 
-struct DecodedSamples {
```

Comment:
moved to pipeline/struct.rs becaus:
- we will need to create those samples from inputs that produce raw data
- I didn't want to make it `pub(crate)`

Reply:
agreed, this needs to be changed